### PR TITLE
validate start date when create sprint

### PR DESCRIPTION
**Pull request by NgocVu1994**

### CODE REVIEW

**Comment by hiennv94** [comment]
ok

File: app/controllers/admin/sprints_controller.rb [hunk 1]
```diff
@@ -3,14 +3,14 @@ class Admin::SprintsController < ApplicationController
...
-  before_action :load_assignee, only: [:new, :edit]
+  before_action :load_assignee, except: :destroy
 
   def create
     if @sprint.save
       flash[:success] = flash_message "created"
       redirect_to admin_project_path(@project)
     else
-      flash[:failed] = flash_messages "not_created"
+      flash[:failed] = flash_message "not_created"
```
Comment (ttuan) [inline app/controllers/admin/sprints_controller.rb]: flash.now

File: app/models/sprint.rb [hunk 2]
```diff
@@ -9,6 +9,9 @@ class Sprint < ActiveRecord::Base
   has_many :log_works
   has_many :master_sprints
 
+  validates :name, presence: true
```
Comment (ttuan) [inline app/models/sprint.rb]: allow_blank: false

**Comment by ttuan** [comment]
OK

**Pull request merged**